### PR TITLE
Added documation for WebView iOS nav bar (optional automation).

### PR DESCRIPTION
**Pull request by unobatbayar**
Added automation documation for WebView iOS setup. (when Navigation bar doesn't appear)

Korean version is not added. (todo)

This MR is optional, I just thought it might help others. I highly value your opinion.

Thank you.

https://github.com/nhn/gpm.unity/issues/329#issuecomment-1366303647

### CODE REVIEW

File: docs/WebView/README.en.md [hunk 1]
```diff
@@ -122,6 +122,27 @@ When performing an iOS build in certain Unity versions, the buttons on **Navigat
...
+// adding -ObjC to Other Linker Flags in Build Settings
+pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");
+```
+ 
+GPMWebView.bundle
+
+```cs
+// adding GPMWebView.bundle to Copy Bundle Resources in Build Phases
+var webViewBundleGuid = pbxProject.AddFile("Frameworks/GPM/WebView/Plugins/IOS/GPMWebView.bundle", "GPMWebView.bundle", PBXSourceTree.Build);  
+pbxProject.AddFileToBuild(appGuid, webViewBundleGuid);
```
Comment (taehyeon-nhn) [inline docs/WebView/README.en.md:143]: When '-ObjC' or 'GPMWebView.bundle' is present, it should be checked to ensure that it works without any problems, or a supplementary description of the content should be provided.

**Review by taehyeon-nhn — COMMENTED** [review]
When '-ObjC' or 'GPMWebView.bundle' is present, it should be checked to ensure that it works without any problems, or a supplementary description of the content should be provided.

**Comment by taehyeon-nhn** [comment]
We will add the Korean version document after approved the request.
Thank you for your pull request.

File: docs/WebView/README.en.md (hunk 1 above)
Comment (unobatbayar) [reply docs/WebView/README.en.md:143]: OK, I will add checking code and test around this then update.

File: docs/WebView/README.en.md [hunk 2]
```diff
@@ -122,6 +122,27 @@ When performing an iOS build in certain Unity versions, the buttons on **Navigat
 
 ![GPMWebViewBundle.png](images/GPMWebViewBundle.png)
 
+#### Optional automation
+
+You can automate above process by using [OnPostprocessBuild](https://docs.unity3d.com/ScriptReference/Build.IPostprocessBuildWithReport.OnPostprocessBuild.html) and [PBXProject](https://docs.unity3d.com/ScriptReference/iOS.Xcode.PBXProject.html).
```
Comment (SangYun-nhn) [inline docs/WebView/README.en.md:127]: Since the contents can depend on the Unity version, why not specify the verified Unity version?

Comment (unobatbayar) [reply docs/WebView/README.en.md:127]: That's a good idea. I'm using Unity 2020.3.21f1, without iOS setup, the navigation buttons are not visible.

I haven't tested on other Unity versions yet, so for now, can I specify this version?

Comment (SangYun-nhn) [reply docs/WebView/README.en.md:127]: Any way would be great as long as users can check the version in the guide
For example :
Note - Checked Unity Version 2020.3.21f1

Comment (unobatbayar) [reply docs/WebView/README.en.md:127]: OK!!

**Review by SangYun-nhn — APPROVED** [review]

File: docs/WebView/README.en.md (hunk 1 above)
Comment (unobatbayar) [reply docs/WebView/README.en.md]: I have checked! They are already checked by Unity and Xcode!

[AddBuildProperty](https://docs.unity3d.com/ScriptReference/iOS.Xcode.PBXProject.AddBuildProperty.html) ignores duplicate values. So only one -ObjC is added!

Same for GPMWebView.bundle, it's not possible to add again programmatically and manually.

To make sure, I have tested with multiple lines:

```cs
                // Add linker
                pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");

                pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");

                pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");

                pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");

                // Add bundle
                var webViewBundleGuid = pbxProject.AddFile("Frameworks/GPM/WebView/Plugins/IOS/GPMWebView.bundle", "GPMWebView.bundle", PBXSourceTree.Build);  
                pbxProject.AddFileToBuild(appGuid, webViewBundleGuid);

                pbxProject.AddFileToBuild(appGuid, webViewBundleGuid);

                pbxProject.AddFileToBuild(appGuid, webViewBundleGuid);
```
Only one'-ObjC' and 'GPMWebView.bundle' were added. There is no error or problems.

I have provided additional description for this. 👍

File: docs/WebView/README.en.md (hunk 2 above)
Comment (unobatbayar) [reply docs/WebView/README.en.md]: It's added! 👍 

Thank you.

**Review by smflt-nhn — APPROVED** [review]

**Review by taehyeon-nhn — APPROVED** [review]

**Pull request merged**